### PR TITLE
Updated testnet for 201 delegates

### PR DESCRIPTION
**Pull request by marilynpereira03**

### CODE REVIEW

File: config.devnet.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
-  "port": 5000,
+  "port": 4002,
   "address": "0.0.0.0",
-  "version": "0.2.0",
+  "version": "1.1.0",
   "fileLogLevel": "info",
   "logFileName": "logs/ark.log",
```
Comment (cyrus19901) [inline config.devnet.json:6]: IT should be BPL.log

File: config.mainnet.json [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+  "ssl": {
+    "enabled": false,
+    "options": {
+      "port": 443,
+      "address": "0.0.0.0",
+      "key": "./ssl/ark.key",
+      "cert": "./ssl/ark.crt"
+    }
+  },
+  "network":"ark",
```
Comment (cyrus19901) [inline config.mainnet.json:218]: The network should be BPL

File: logic/block.js [hunk 3]
```diff
@@ -21,16 +23,16 @@ function Block (scope, cb) {
 // Private methods
 __private.blockReward = new BlockReward();
 
-__private.getAddressByPublicKey = function (publicKey) {
-	return arkjs.crypto.getAddress(publicKey);
+__private.getAddressByPublicKey = function (publicKey, network) {
+	return arkjs.crypto.getAddress(publicKey, network.pubKeyHash);
```
Comment (cyrus19901) [inline logic/block.js:27]: Can we use BPL-js instead?

File: package.json [hunk 4]
```diff
@@ -5,17 +5,19 @@
...
     "start:test": "node app.js --genesis test/genesisBlock.json --config test/config.json",
-    "start:newtest": "node app.js --genesis tasks/genesisBlock.json --config tasks/config.json",
+    "start:bpldevnet": "node app.js --genesis private/genesisBlock.private.json --config private/config.private.json",
     "test": "./node_modules/.bin/mocha",
     "cov": "./node_modules/.bin/mocha --require blanket -R html-cov test/helpers test/logic > tmp/coverage.html"
   },
   "author": "Boris Povod <boris@crypti.me>, Pavel Nekrasov <landgraf.paul@gmail.com>, Oliver Beddows <oliver@lisk.io>, FX Thoorens <fx.thoorens@ark.io>",
   "dependencies": {
-    "arkjs": "https://github.com/arkecosystem/ark-js#newtestnet",
+    "arkjs": "github:arkecosystem/ark-js#mainnet",
```
Comment (cyrus19901) [inline package.json:18]: need to add the dependency for bpl-js

File: tasks/createGenesisBlockOneVote.js [hunk 5]
```diff
@@ -6,21 +6,27 @@ var crypto = require('crypto');
...
+    "version": "1.1.0",
     "fileLogLevel": "info",
     "logFileName": "logs/ark.log",
     "consoleLogLevel": "debug",
     "trustProxy": false,
     "db": {
         "host": "localhost",
         "port": 5432,
-        "database": "ark_test",
+        "database": "ark_devnet",
```
Comment (cyrus19901) [inline tasks/createGenesisBlockOneVote.js:29]: Can we change the name to BPL-Devnet

**Review by locohammerhead — CHANGES_REQUESTED** [review]
Line 10 is calling ark.js
should be BPL.js
line 101 we are calling networks.ark

**Pull request merged**